plutus-metatheory/src/Algorithmic/CEKC.lagda.md

```diff
@@ -86,7 +88,7 @@ BUILTIN lessThanEqualsInteger σ ((_ ,, _ ,, V-con (integer i) ,, _) ,, (_ ,, _
 BUILTIN greaterThanInteger σ ((_ ,, _ ,, V-con (integer i) ,, _) ,, (_ ,, _ ,, V-con (integer i') ,, _) ,, tt) = _ ,, decIf (i Builtin.Constant.Type.>? i') (con (bool true)) (con (bool false)) ,, []
 BUILTIN greaterThanEqualsInteger σ ((_ ,, _ ,, V-con (integer i) ,, _) ,, (_ ,, _ ,, V-con (integer i') ,, _) ,, tt) = _ ,, decIf (i ≥? i') (con (bool true)) (con (bool false)) ,, []
 BUILTIN equalsInteger σ ((_ ,, _ ,, V-con (integer i) ,, _) ,, (_ ,, _ ,, V-con (integer i') ,, _) ,, tt) = _ ,, decIf (i ≟ i') (con (bool true)) (con (bool false)) ,, []
-BUILTIN concatenate σ ((_ ,, _ ,, V-con (bytestring b) ,, _) ,, (_ ,, _ ,, V-con (bytestring b') ,, _) ,, tt) = _ ,, con (bytestring (append b b')) ,, []
+BUILTIN concatenate σ ((_ ,, _ ,, V-con (bytestring b) ,, _) ,, (_ ,, _ ,, V-con (bytestring b') ,, _) ,, tt) = _ ,, con (bytestring (concat b b')) ,, []
```

Comment:
@jmchapman I renamed what was `append` to `concat`, because

1. we now also have an `append` built-in function
2. `concat` corresponds to `concatenate` better